Header header
int8 drone_id
int8 task_id
float32 score
